easybuild/easyconfigs/p/PRIMME/PRIMME-3.2-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'ConfigureMake'
+
+name = 'PRIMME'
+version = '3.2'
+
+homepage = 'http://www.cs.wm.edu/~andreas/software/'
+description = """PRIMME, pronounced as prime, is a high-performance library for computing a few eigenvalues/eigenvectors, and singular values/vectors.
+PRIMME is especially optimized for large, difficult problems.
+Real symmetric and complex Hermitian problems, standard A x = \lambda x and generalized A x = \lambda B x, are supported.
+Besides, standard eigenvalue problems with a normal matrix are supported.
+It can find largest, smallest, or interior singular/eigenvalues, and can use preconditioning to accelerate convergence.
+"""
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('SciPy-Stack', '2023a'),
+]
+multi_deps = {'Python': ['3.8', '3.9', '3.10']} #Python extention not compatible with Python 3.11
+
+source_urls = [
+    'https://github.com/primme/primme/archive/refs/tags/',
+]
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    '8ff242a356cea465c9728a26cb6e0487712d9ae51050a362de487e3b13a2fe9b', #sources
+]
+
+# No configure
+skipsteps = ['configure']
+
+# Build static and shared libraries
+build_cmd = "make lib && make solib"
+
+# Custom install because target "make install" does not work
+# Note we can't use the "MakeCp" easyblock because we use installopts
+# to install the Python extention
+install_cmd = " && ".join([
+    "cd %(builddir)s/primme-%(version)s",
+    "cp -r lib %(installdir)s/lib",
+    "cp -r include %(installdir)s/include",
+])
+
+# Install the Python extension
+installopts = ' && '.join([
+    "",
+    "cd %(builddir)s/primme-%(version)s/Python",
+    "sed -i 's/scipy\.sparse\.linalg\.interface/scipy\.sparse\.linalg/g' primme.c primme.pyx", #Fix a DepreciationWarning
```

Comment:
Shouldn,t then use a different scipy version? Instead of changing the interface? 
This has the potential to create issues where the code was developped for an older interface and we substitute for a newer one without fixing the whole code.

Reply:
We can use scipy-stack/2021a, but this is an old version and we would just have the interface for Python 3.7 to 3.9. Yet, only the `aslinearoperator` function from `scipy.sparse.linalg.interface` namespace is used, which in fact, import `aslinearoperator` from `scipy.sparse.linalg` namespace (see https://github.com/scipy/scipy/blob/main/scipy/sparse/linalg/interface.py and https://github.com/scipy/scipy/blob/main/scipy/_lib/deprecation.py).